share/spice/public_holidays/record.handlebars

```diff
@@ -3,8 +3,8 @@
         {{#each holidays}}
             <tr class="record__row  {{#if ../meta.options.rowHighlight}}record__row--highlight{{/if}}">
                 <td class="record__cell  record__cell--key">{{date}}</td>                
-                <td class="record__cell  record__cell--key">{{name}}</td>                
-                <td class="record__cell  record__cell--key">{{notes}}</td>                
+                <td class="record__cell  record__cell--value">{{name}}</td>
+                <td class="record__cell  record__cell--value">{{notes}}</td>
```

Comment:
@moollaza Tried your suggestion but the layout is a little inconsistent afterwards

![optional-notes](https://cloud.githubusercontent.com/assets/16732873/13830224/388c3e48-ebc3-11e5-88a3-31b4ab20e1ee.png)